manifests/0000_90_kube-controller-manager-operator_05_alert-pdb.yaml

```diff
@@ -11,17 +11,17 @@ spec:
       rules:
         - alert: PodDisruptionBudgetAtLimit
           annotations:
-            message: The pod disruption budget is preventing further disruption to pods because it is at the minimum allowed level.
+            message: The pod disruption budget {{ $labels.namespace }}/{{ $labels.poddisruptionbudget }} is preventing further disruption to pods because it is at the minimum allowed level.
```

Comment:
We don't support variables in our rendering pipeline, yet so I'd suggest dropping this. When there will be a need we'll add them. For now it'll be simpler w/o those and will clear the unit failure.

Reply:
Thanks for the info, I wasn't aware that the manifests could be templatized. I'll rollback to the previous version.
The templating here was meant for the Prometheus server when it evaluates the alerting rule. See the Prometheus [documentation](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/#templating).